_docs/tasks/traffic-management/ingress.md

```diff
@@ -269,6 +269,204 @@ rules.
    curl -I -k https://$INGRESS_HOST/status/200
    ```
 
+1. Configuring RBAC for ingress key/cert.
+
+   There are service accounts which can access this ingress key/cert, and this leads to risks of 
+   leaking key/cert. We can set up Role-Based Access Control ("RBAC") to protect it.
+   [istio/install/kubernetes/istio.yaml](https://github.com/istio/istio/blob/master/install/kubernetes/istio.yaml)
+   defines ClusterRoles and ClusterRoleBindings which allow service accounts in namespace istio-system
+   to access all secret resources. We need to update or replace these RBAC set up to only allow
+   istio-ingress-service-account to access ingress key/cert.
+
+   We can use kubectl to list all secrets in namespace istio-system that we need to protect using RBAC.
+   
+   ```bash
+   kubectl get secrets -n istio-system
+   ```
+   This produces the following output:
+   
+   ```bash
+   NAME                                        TYPE                                  DATA      AGE
+   istio-ca-secret                             istio.io/ca-root                      2         7d
+   istio-ingress-certs                         kubernetes.io/tls                     2         7d
+   istio-ingress-service-account-token-kwr85   kubernetes.io/service-account-token   3         7d
+   istio.istio-ingress-service-account         istio.io/key-and-cert                 3         7d
+   ......
+   ```
+   
+1. Update RBAC set up for istio-pilot-service-account and istio-mixer-istio-service-account.
+   
+   Delete existing ClusterRoleBindings and ClusterRole.  
+
+   ```bash
+   kubectl delete ClusterRoleBinding istio-pilot-admin-role-binding-istio-system
+   kubectl delete ClusterRoleBinding istio-mixer-admin-role-binding-istio-system
+   kubectl delete ClusterRole istio-mixer-istio-system
```

Comment:
Is there a similar role for istio-pilot-istio-system?

Reply:
Is there any way to recover the deletion here at the end of this file? The idea is to clean up the change in this file after everything is done.

Reply:
And what privileges will pilot/mixer lose after this change? I am afraid this will break existing system. Let me add some pilot and mixer guy.

Reply:
istio-pilot-istio-system is also bound to account istio-ingress-service-account by istio-ingress-admin-role-binding-istio-system. We will delete istio-pilot-istio-system in next step, after deleting that ClusterRoleBinding object.

According to https://github.com/istio/istio/blob/11dc2fb937a23157215fffa06921cf507d27f757/install/kubernetes/istio.yaml#L22
istio-pilot-istio-system grants access permission to many resources. We need to bind those rules to istio-pilot-service-account. We should apply similar role binding process to account istio-mixer-service-account and  istio-ca-service-account